index.bs

```diff
@@ -10766,11 +10766,11 @@ and is described in more detail in [[#interface-prototype-object]].
 Note: Since an [=interface object=] is a [=function object=]
 the <code>typeof</code> operator will return "function" when applied to an interface object.
 
-<div algorithm="to create an interface object">
+<div algorithm>
 
-    The [=interface object=]
-    for a given non-callback [=interface=] |I| with [=identifier=] |id|
-    and in [=Realm=] |realm| is created as follows:
+    The [=interface object=] for a given non-callback [=interface=] |I|
+    with [=identifier=] |id| and in [=Realm=] |realm|
+    is <dfn lt="create an interface object">created</dfn> as follows:
```

Comment:
Most of these algorithms don't return anything. Although I don't think they have callers yet, probably they should return the thing they create.

Reply:
Fixed in https://github.com/heycam/webidl/pull/494/commits/37ae0fb19dc2465ea20cb88adb746729b7dc6af7.